ampd-handlers/Dockerfile

```
@@ -1,10 +1,13 @@
 FROM rust:1.81-bookworm as builder
+RUN apt-get update && apt-get install -y clang protobuf-compiler cmake
 
-WORKDIR /ampd-handlers
+WORKDIR /app
 COPY . .
-RUN cargo build --release --bin ampd_handler
 
-FROM debian:bullseye-slim
+# Build the ampd-handlers binary
+RUN cargo build --release --bin ampd-handlers
+
+FROM debian:bookworm-slim
```

```suggestion
FROM debian:bookworm-slim as runner
```
I don't it doesn't do anything, but just better for readability